Libraries/CustomComponents/NavigationExperimental/NavigationCard.js

```diff
@@ -87,6 +87,8 @@ class NavigationCard extends React.Component<any, Props, any> {
       ...props,
     } = this.props;
 
+    const defaultStyle = NavigationCardStackStyleInterpolator.forHorizontal(props);
```

Comment:
I agree with @satya164 

The designer will asks for 100+ style of back button. There isn't really a generic back button except the one that can only be used in simple example.